tests/modules/nf-core/ataqv/ataqv/main.nf

```diff
@@ -51,7 +51,7 @@ workflow test_ataqv_ataqv_tss {
     ] 
     tss_file = file(params.test_data['sarscov2']['genome']['test_bed'], checkIfExists: true)
    
-    ATAQV_ATAQV ( input, 'human', tss_file, [], [] )
+    ATAQV_ATAQV ( input, 'human', [], tss_file, [], [] )
```

Comment:
Seeing as it's a string can we pass an empty string instead?

Reply:
Both will work. The string is probably a little cleaner
.

Reply:
The parameter might resolve to `null` in cases where it is not provided, as that's what the `getGenomeAttribute` when the parameter is not found. Given that, I think that `null` is a more representative "default" to what we'll see in the wild.

Reply:
At this point, the module can be used in any workflow independent of that function so I think it makes more sense to default to the type specified in the `meta.yml`. Especially since these tests are unrelated to how the module is used in the workflow.

Reply:
Fair. Fixed

Reply:
But we do default empty strings to `null` in the pipeline `nextflow.config` so you are right too 😅 In these scenarios I just find it easier to look at the tests and know what type the input channel is likely to be if we default to the native type, especially when you have loads of inputs.